src/main/webapp/assets-lti/js/app/app.js

```diff
@@ -7,6 +7,7 @@ canvasSupportApp.run(function($rootScope) {
   //for any init values needed
   $rootScope.server = '';
   $rootScope.user = {};
+  $rootScope.csv_throttle = 1000;
   //to turn on/off the 2 functionalities
```

Comment:
We can make this a property, backend will send the value along with other context parameters